code/api/src/migrations/6-user-image.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('users', 'image', {
+      type: Sequelize.STRING,
+      defaultValue: "https://images.unsplash.com/photo-1537151608828-ea2b11777ee8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=939&q=80"
```

Comment:
Thanks for adding the defaultValue image!

Reply:
Will we change the defaultValue property when we change the image or will we create a new image property?